entry/src/main/ets/pages/index.ets

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (c) 2022 Application Library Engineering Group.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { MaterialBanner, BannerType, BannerModel } from '@ohos/material-banner'
+
+@Entry
+@Component
+struct Index {
+  @State image: Resource= $r('app.media.noc')
+  @State display1: boolean = false
+  @State display2: boolean = false
+  @State display3: boolean = false
+  private bannerModel1: BannerModel= new BannerModel(BannerType.OneButton)
+  private bannerModel2: BannerModel= new BannerModel(BannerType.TwoButton)
+  private bannerModel3: BannerModel= new BannerModel(BannerType.ImageButton)
+
+  aboutToAppear() {
+    this.bannerModel1.setBannerMessage("Your Device is Offline");
```

Comment:
Place all the strings at string.json and add localization also for chinese language (can translate strings online and place)